src/site/markdown/server/install/guice-cassandra.md

```diff
@@ -25,7 +25,7 @@ mvn clean install
 
 ### Requirements
 
- * Cassandra 3.11.3
+ * Cassandra 3.11.10
```

Comment:
If we change the requirement, it means we don't support 3.11.3 any longer and thus we need a breaking change entry, right?